activestorage/active.py

```diff
@@ -43,10 +79,13 @@ def __init__(self, uri, ncvar, missing_value=None, _FillValue=None, valid_min=No
         """
         # Assume NetCDF4 for now
         self.uri = uri
-        if self.uri is None:
-            raise ValueError(f"Must use a valid file for uri. Got {self.uri}")
-        if not os.path.isfile(self.uri):
-            raise ValueError(f"Must use existing file for uri. {self.uri} not found")
+        self.do_active = True
+        # run in active storage mode only if the uri analyzer returns True
+        uri_analyzer_result = uri_analyzer(self.uri)
+        self.do_active = uri_analyzer_result
+        # otherwise just get out and return active.do_active = False
+        if not self.do_active:
```

Comment:
Hi V - I thought that if `do_active` is False then we do not want to return, rather we want to set `_version` to `2` (or is it `1`) ...

Reply:
yeps indeed, I followed up on your change request, thanks, bud!

Reply:
Hi V, I don't get what `uri_analyzer` is doing. If the file doesn't exist, then we can do anything, so setting `self._version = 2` won't help us. 

From a cf-python perspective, I know that the file exists  - the question is "is it in location that supports active storage requests?" If "yes" then `self._version = 1`, and if "no" then `self._version = 2`